tests/unit/ConsentContinuationOut.spec.ts

```diff
@@ -8,17 +8,57 @@ import { ConfirmDialog, PaymentErrorDialog, ResumeErrorDialog, SaveErrorDialog,
 import { Certify, DetailComment } from '@/components/common'
 import { CourtOrderPoa } from '@bcrs-shared-components/court-order-poa'
 import { DocumentDelivery } from '@bcrs-shared-components/document-delivery'
+import axios from '@/axios-auth'
+import flushPromises from 'flush-promises'
 import mockRouter from './mockRouter'
+import sinon from 'sinon'
 import VueRouter from 'vue-router'
 
+// suppress various warnings:
+// - "Unknown custom element <affix>" warnings
+// - "$listeners is readonly"
+// - "Avoid mutating a prop directly"
+// ref: https://github.com/vuejs/vue-test-utils/issues/532
+Vue.config.silent = true
+
 Vue.use(Vuetify)
 
 const store = getVuexStore() as any // remove typings for unit tests
 const localVue = createLocalVue()
 localVue.use(VueRouter)
 
 describe('Consent to Continuation Out view', () => {
+  let sinonAxiosGet: any
+
   beforeEach(() => {
+    sinonAxiosGet = sinon.stub(axios, 'get')
```

Comment:
If you just have 1 mock getter then you don't need to assign it to a variable -- just just it directly on line 39.

(see my next comment)